scripts/superdesk/editor2/add-embed.ctrl.js

```diff
@@ -71,9 +71,13 @@ EMBED_PROVIDERS, $scope, editor, config, $injector) {
                         name: EMBED_PROVIDERS.youtube
                     }
                 ];
-                // prepend with custom handlers from config
-                if (config.editorEmbedCodeParsers) {
-                    knownProviders = $injector.invoke(config.editorEmbedCodeParsers).concat(knownProviders);
```

Comment:
I just merged https://github.com/superdesk/superdesk-core/pull/508, so you can now add `apps.vidible` to the installed app in this branch